fastlane/helpers/android_promo_screenshot_helper.rb

```diff
@@ -44,6 +47,8 @@ def self.require_font()
         Dir.mkdir(font_folder) unless File.exist?(font_folder)
         Fastlane::Actions::sh("wget \"https://fonts.google.com/download?family=Noto%20Serif\" -O \"#{font_folder}/noto.zip\"")
         Fastlane::Actions::sh("unzip \"#{font_folder}/noto.zip\" -d \"#{font_folder}\"")
+        Fastlane::Actions::sh("wget \"https://fonts.google.com/download?family=Roboto\" -O \"#{font_folder}/roboto.zip\"")
+        Fastlane::Actions::sh("unzip \"#{font_folder}/roboto.zip\" -d \"#{font_folder}\"")
```

Comment:
You need to install the fonts on your system after download otherwise it won't be able to use them